Cargo.toml

```diff
@@ -21,20 +21,20 @@ num-traits = "0.2"
 oauth2 = "4"
 reqwest = "0.11"
 serde = { version = "1", features = ["derive"] }
+tokio = { version = "1", features = ["sync"] }
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
-yew = "0.19"
-yew-agent = "0.1"
+yew = "0.20"
```

Comment:
I think you need to put "0.20.0", This is what I have in my yew app to get renderer to work. `yew = { version = "0.20.0", features = ["csr"] }`

Reply:
True, you need that for an app, but not for a library.

Reply:
Ah, makes sense!